test.js

```diff
@@ -100,6 +100,13 @@ it('should preserve property order', function () {
 	assert.equal(Object.keys(target).join(''), letters);
 });
 
+it('target can be primatives', function () {
```

Comment:
Although I'd rather see AVA as testing framework, if `mocha` is used, the convention of titles is more like this:

```
it('should accept primitives as target', function () {
```